### ISSUE TITLE
[cli] Add Create User and Get Users to CLI

### ISSUE THREAD

**Issue by sumerjabri** [craftercms/craftercms#5249]
#### Is your feature request related to a problem? Please describe.
CLI doesn't support creating a user or users.

#### Describe the solution you'd like
Add create a user call. As an added bonus, have it take a list of users (CSV?) to bulk create users.
Add get users and have that export (CSV?).

#### Describe alternatives you've considered
{{A clear and concise description of any alternative solutions or features you've considered.}}

**Comment by phuongnq** [comment]
Added PR for `create-user` CLI. Sample of usage:

For single user from command parameters:

```
$ ./crafter-cli create-user -un phil.nguyen -pw "abc123#456(ABC123" -fn Phil -ln Nguyen -em phil.nguyen@craftercms.com -e local                   
Created. User: phil.nguyen

```

Bulk from a CSV:
```
$ ./crafter-cli create-user -bf /home/pnguyen/Desktop/users.csv -e local                     
Created. User: phil.nguyen1
Created. User: phil.nguyen2
Created. User: phil.nguyen3
```

Sample CSV: (need to have headers with value such as `username,password,firstName,lastName,email,enabled,externallyManaged`)
[users.csv](https://github.com/craftercms/craftercms/files/13770008/users.csv)

**Comment by phuongnq** [comment]
@sumerjabri For the list users API, does it have the same parameters as the Studio API here: https://docs.craftercms.org/en/4.1/_static/api/studio.html#tag/users/operation/getUsers

```
site_id	
string
The site ID to filter users for a particular site

keyword	
string
The keyword to filter users

offset	
integer <int32>
Offset of first record in the response

limit	
integer <int32>
Number of records to return

sort	
string
The fields to use for sorting, plus the asc or desc keyword case-insensitive. Multiple fields are separated by commas. Example: column1 ASC, column2 DESC.
```

**Comment by Euquimides** [comment]
Hi! Feature verified implemented for v4.2. Thank you! Closing this ticket.
